src/components/shared/modals/IssueTokenModal.tsx

```diff
@@ -59,7 +59,7 @@ export default function IssueTokenModal({
       transactionPending={transactionPending}
     >
       <p>
-        {!Boolean(isNewDeploy) ? (
+        {!isNewDeploy ? (
```

Comment:
> This wasn't in the initial warning, but it was showing up. It might have just been a stale yarn cache that didn't catch this rule the first time because my bundle/lint changed so dramatically between branches.

Same here.